From b0df9e7771ac9148004187bb53ba13ac40b01cae Mon Sep 17 00:00:00 2001
Message-Id: <b0df9e7771ac9148004187bb53ba13ac40b01cae.1316291516.git.kevin.kofler@chello.at>
From: Kevin Kofler <kevin.kofler@chello.at>
Date: Sat, 17 Sep 2011 18:18:24 +0200
Subject: [PATCH] K3b::IsOverburnAllowed: Increase tolerance from 10% to 25%.

10% are not enough to allow using CD-R90 and CD-R99 media to full capacity.
The new 25% tolerance allows using those media while still catching the common
case of DVD ISO vs. CD media.

BUG: 276002
REVIEW: 102648
---
 libk3b/core/k3bglobals.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libk3b/core/k3bglobals.cpp b/libk3b/core/k3bglobals.cpp
index d66f601..28c12c6 100644
--- a/libk3b/core/k3bglobals.cpp
+++ b/libk3b/core/k3bglobals.cpp
@@ -595,7 +595,7 @@ bool K3b::IsOverburnAllowed( const K3b::Msf& projectSize, const K3b::Msf& capaci
 bool K3b::IsOverburnAllowed( const Msf& projectSize, const Msf& capacity, const Msf& usedCapacity )
 {
     return( k3bcore->globalSettings()->overburn() &&
-        (projectSize + usedCapacity) <= ( capacity.lba() - usedCapacity.lba() + capacity.lba() * 10 / 100 ) ); // 10% tolerance in overburn mode
+        (projectSize + usedCapacity) <= ( capacity.lba() - usedCapacity.lba() + capacity.lba() / 4 ) ); // 25% tolerance in overburn mode
 }
 
 
-- 
1.7.6

